packages/hash/api/src/model/knowledge/link.model.ts

```diff
@@ -401,12 +410,13 @@ export default class {
   /**
    * Remove the link.
    *
-   * @param removedbyId - the id of the user removing the link
+   * @param params.removedById - the id of the user removing the link
```

Comment:
```suggestion
   * @param params.removedById - the id of the account removing the link
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/1197/commits/51c3a539be68ca7bf1c7ffa8f13550e6f20efee4